components/scream/src/physics/p3/p3_functions.hpp

```diff
@@ -237,6 +237,19 @@ struct Functions
   KOKKOS_INLINE_FUNCTION
   static Spack qv_sat(const Spack& t_atm, const Spack& p_atm, const bool ice);
 
+  KOKKOS_FUNCTION
+  static void cloud_water_conservation(const Spack& qc, const Spack& qcnuc,const Scalar dt,
```

Comment:
This is the main place to insert comments about what the routine does, if I've been following Jim's pattern correctly.